.devcontainer/postCreate.sh

```diff
@@ -6,4 +6,4 @@ pip install poetry
 
 poetry install --with dev
 
-poetry run pre-commit install
```

Comment:
Sounds petty - but can we add the new line back in. Newlines are useful at the bottom of files